portfolio/src/main/webapp/script.js

```diff
@@ -92,4 +94,35 @@ function highlightSection(selectedIdString) {
         section.style.border = 'none'; 
       }
     }); 
+}
+
+/**
+ * This function sends a request to the /data url and fetches the text from there.
+ * Then, the function puts the received text into the quoteWrapper, which is located 
+ * below the bulletin points on the page. 
+ */
+function getQuotes() {
+  fetch('/data')
+      .then(response => response.json())
+      .then((quotesJson) => {
+        const quoteWrapper = document.getElementById('quoteWrapper');
+        quoteWrapper.innerHTML = '';
+
+        Object.keys(quotesJson).forEach(
```

Comment:
Instead of looping over the keys, just to pull out the value, maybe we should just loop over the values?

Reply:
Yes that would be better. I didn't realize that we can loop over the values directly.